src/bonsai/rest/leverage.ts

```diff
@@ -43,62 +32,21 @@ export function setUpUserLeverageParamsQuery(store: RootStore) {
           },
           {} as { [clobPairId: number]: string }
         );
-        const clobPairToSubaccountNumber = childSubaccounts.reduce(
-          (acc, sa) => {
-            return {
-              ...acc,
-              ...Object.values(sa.openPerpetualPositions).reduce(
-                (positionAcc, op) => {
-                  const market = allMarkets[op.market];
-                  if (market === undefined) {
-                    return positionAcc;
-                  }
-                  return {
-                    ...positionAcc,
-                    [market.clobPairId]: op.subaccountNumber,
-                  };
-                },
-                {} as typeof acc
-              ),
-            };
-          },
-          {} as { [clobPairId: number]: number }
-        );
 
-        const subaccountLeverages = await Promise.all(
-          childSubaccounts.map(async (sa) => {
-            return compositeClient.validatorClient.get.getPerpetualMarketsLeverage(
-              parent.address,
-              sa.subaccountNumber
-            );
-          })
+        const leverages = await compositeClient.validatorClient.get.getPerpetualMarketsLeverage(
+          data.parentSubaccount!.address,
+          0
```

Comment:
can replace `0` with parentSubaccountInfo.subaccount after you make change to the selector used. we should avoid hardcoding `0` everywhere and rely on the selector (even though the selector hardcodes it, it is at least done in one place)